vignettes/design-principles.Rmd

```diff
@@ -24,25 +24,40 @@ None of the sections are required, feel free to remove any sections not relevant
+- Researchers who would like to compare new estimates to existing parameters.
+- Individuals who would like to generate a report of the parameters in the database, including key summary statistics and plots for a report.
+  
+## Minimal interface
 
-## Input/Output/Interoperability
+- Sufficient summary statistics: show multiple studies at the same time. 
+- A comparison of available distributions for the same disease, allowing user interactivity by selecting citations to visualise.
+- Ability to highlight/select individual lines on the plot and see its associated information.
+- Context from which parameters were obtained (available in `{epireview}`): community-based, health care workers, etc.
+- An option to download a report containing the code used to generate the dashboard visuals, and information on the disease, delay distribution, and other associated information (citation, etc).
+- Provide an easy way to copy parameter estimates and citation.
 
-< Describe the data structures (i.e. vectors, `<data.frames>` or classes) that are given as input to the key functions and what data structures the functions return. The design decisions around these I/O choices could also mention how it enhances interoperability with other R packages or pipelines (e.g. with `%>%`). >
+### Out of scope
 
+- Impact of different parameters on downstream estimates of $Rt$, $CFR$, etc.
 
 ## Design decisions
 
-< A list of bullet points each explaining a design decision and its reasoning. >
+## Input/Output/Interoperability
+
+- `{epiparameterDashboard}` will read in the data provided by `{epiparameter}` and `{epireview}` as `<data.frame>` and output the dashboard that provides options to visualise parameters from the two packages separately and compare values across the two packages.
 
 ## Dependencies
 
-< A list of dependencies used by the package with some explanation as to why they are required. Not all dependencies need to be explained and it is best to explain the key dependencies. It can be used to give context to why certain dependencies are used (e.g. "This package is expected to be used in tidyverse pipelines and as such, we consider these tidyverse packages good dependencies that will already be installed on a user's computer."). This section can also mention dependencies that are planned to be removed or added in future development. Suggested dependencies do not need to be explained unless they are unusual and may surprise developers with their inclusion. >
-
-## Development journey
+- The ability to download static output, for example, a report, will determine whether to use `{flexdashboard}` vs `shiny`.
 
-<  If the package has undergone any large refactoring this section can be used to explain the changes. >
+- plotly (for interactivity with plots)
+- Shiny (for dashboard)
+- flexdashboard (for dashboard)
+- epiparameter (for data and functions)
+- epireview (for data and functions)
+- DT (for tables)
```

Comment:
Noting down here that [`{contructive}`](https://cynkra.github.io/constructive/) may be useful for printing the code that was used to produce output.